### ISSUE TITLE
Poetry 1.6.1

### ISSUE THREAD

**Issue by ClaasJG** [fedej/poetry-grpc-plugin#54]
Hi,

poetry-grpc-plugin requires poetry (>=1.2.0,<1.6) but [poetry 1.6.1](https://python-poetry.org/history/) got released 4 days ago.
As far as I am aware, the plugin will still work with poetry 1.6.1 and the fix should be as easy as changing the version range.

Have a nice weekend

ClaasJG

```
poetry self add poetry-grpc-plugin
Using version ^0.1.7 for poetry-grpc-plugin

Updating dependencies
Resolving dependencies... (0.2s)

Because no versions of poetry-grpc-plugin match >0.1.7,<0.2.0
 and poetry-grpc-plugin (0.1.7) depends on poetry (>=1.2.0,<1.6), poetry-grpc-plugin (>=0.1.7,<0.2.0) requires poetry (>=1.2.0,<1.6).
So, because poetry-instance depends on both poetry (1.6.1) and poetry-grpc-plugin (^0.1.7), version solving failed.
```

**Comment by xiezhuojin** [comment]
Same thing happened.

**Comment by wkedz** [comment]
Hi, I have the same issue as ClaasJG. 

All dependencies are functioning correctly with Poetry 1.6.1. Are there any plans to update the version range in the near future?

Best regards.

**Comment by fedej** [comment]
Apologies for the delay, I no longer have spare time to actively maintain this project. 

I just released 0.1.8, which allows any 1.x version of poetry. I'm accepting that poetry minor version releases that change certain internals can break this plugin.

Closing this issue, please reopen it if you continue to have issues.
